communities/genome/lab/assembly.yml

```diff
@@ -38,7 +38,7 @@ tabs:
           - label: HiC reads mapped to assembly <code>mapped_reads.bam</code>
             datatypes:
               - bam
-        button_link: "{{ galaxy_base_url }}/tool_runner?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2yahs"
+        button_link: "button_link: "{{ galaxy_base_url }}?tool_id=toolshed.g2.bx.psu.edu%2Frepos%2Fiuc%2Fyahs%2Fyahs"
```

Comment:
Yes, looks correct but it seems like you don't actually need the URL encoding. This also works just fine:
https://genome.usegalaxy.org.au/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/yahs/yahs

Reply:
And would lead to less typos!